incubator/airflow/templates/deployments-flower.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "airflow.fullname" . }}-flower
+  labels:
+    app: {{ template "airflow.name" . }}-flower
+    chart: {{ template "airflow.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.airflow.flower_replicas }}
```

Comment:
Does it make sense to have more that one flower server. Wouldnt a single celery UI suffice ?

Reply:
Never experienced this limitation, but why not. I can remove it.